### PR TITLE
add Cassandra kind

### PR DESCRIPTION
**Pull request by byashimov**
Adds Cassandra Kind

### CODE REVIEW

File: controllers/cassandra_controller.go [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+	return &a.Status
+}
+
+func (a *cassandraAdapter) getServiceCommonSpec() *v1alpha1.ServiceCommonSpec {
+	return &a.Spec.ServiceCommonSpec
+}
+
+func (a *cassandraAdapter) getUserConfig() any {
+	return &a.Spec.UserConfig
+}
```
Comment (byashimov) [inline controllers/cassandra_controller.go:67]: @ivan-savciuc `Spec` is unique per Kind, so only `Status` and `ObjectMeta` are sharable. So it doesn't help much, most of the methods has to be customised.

File: controllers/generic_service_handler.go [hunk 2]
```diff
@@ -120,9 +121,12 @@ func (h *genericServiceHandler) get(a *aiven.Client, object client.Object) (*cor
...
 
 		metav1.SetMetaDataAnnotation(o.getObjectMeta(), instanceIsRunningAnnotation, "true")
-	}
 
-	return o.newSecret(s), nil
+		// Some services get secrets after they are running only,
+		// like ip addresses (hosts)
+		return o.newSecret(s), nil
+	}
+	return nil, nil
```
Comment (byashimov) [inline controllers/generic_service_handler.go:129]: @ivan-savciuc this change changes a lot actually.

**Review by Serpentiel — APPROVED** [review]
lgtm, good stuff

**Comment by byashimov** [comment]
> lgtm, good stuff

Sorry, I forgot to update the Changelog :(

**Review by Serpentiel — APPROVED** [review]
lgtm

File: main.go [hunk 3]
```diff
@@ -32,6 +32,7 @@ const port = 9443
 func init() {
 	utilruntime.Must(clientgoscheme.AddToScheme(scheme))
 	utilruntime.Must(v1alpha1.AddToScheme(scheme))
+	utilruntime.Must(v1alpha1.AddToScheme(scheme))
```
Comment (ivan-savciuc) [inline main.go:35]: Please remove this duplicate line

**Review by ivan-savciuc — CHANGES_REQUESTED** [review]
Just one tiny thing and we are good to go!

**Review by ivan-savciuc — APPROVED** [review]
LGTM

**Pull request merged**